iset.mm

```diff
@@ -115790,6 +115790,432 @@ Infinite sums (cont.)
     VGVDCVDUEFZEFZVDVDEFCVDKJVDUFUGZCQOVGVKRSVLVDCQVDUHJNVDUIOVLVDRUJUKVDUMTULU
     NVDAUOTVJVDVDEUPUQVDSCDURLUSPUTVAVBVC $.
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Ratio test for infinite series convergence
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d A m p $.  $d m ph $.
+    cvgratnnlembern.3 $e |- ( ph -> A e. RR ) $.
+    cvgratnnlembern.4 $e |- ( ph -> A < 1 ) $.
+    cvgratnnlembern.gt0 $e |- ( ph -> 0 < A ) $.
+    $( Lemma for ~ cvgratnn .  The rate at which a constant less than one
+       raised to a power shinks as the exponent increases.  The answer is in
```

Comment:
shinks?

Reply:
Probably "shrinks" was meant, although I don't see why not "decreases". In any case, this is not what the statement is about.
Since p can be made explicit, why not prove the statement with an explicit expression instead of `E. p` ?

Reply:
"decreases" does seem better but I'm not sure I quite follow "this is not what the statement is about". Is the statement inaccurate or is there just a better way to express the idea?

> Since p can be made explicit, why not prove the statement with an explicit expression instead of E. p ?

Hmm, if memory serves at the time I formulated this I was thinking that `p` would somehow be based on `arch` which isn't readily amenable to being replaced by ceiling in iset.mm.

But between then and when I finished it looks like `p` got a nice closed form, namely `(1 / ((1 / 𝐴) − 1))`. I guess I should probably go back and re-do this. That'll affect this lemma and cvgratnnlemfm and cvgratnnlemrate, so it is probably a few days work. Unless people are eager to merge something now (and submit an issue for the cleanup), I suppose I'll leave this pull request open until I can revise it.

Reply:
I haven't looked carefully, but the statement is not a claim about something being decreasing.

From your comment, I thought that you had an expression for $p$, so I think keeping the explicit result is better. I'm willing to wait !

Reply:
> I haven't looked carefully, but the statement is not a claim about something being decreasing.

Since you are waiting for the other thing, you have time to either look more carefully or figure we won't worry about it. The concept we are trying to talk about is what happens to `A ^ m` as `m` gets larger (in a setting where we care about how fast that happens, or modulus of convergence if we want to talk about it that way).

Reply:
You're studying f(m) := A^m.
The statement says that for m large enough, one has f(m) < p/m.
The current comment is hard to parse, but seems to say that f(m) is decreasing.
These are two different things.

Maybe you can just write:
> Upper bound for a geometric progression of positive ratio less than one.

Reply:
I could elaborate more on what that comment was trying to get at but it is a bit irrelevant because I agree that "Upper bound for a geometric progression of positive ratio less than one." more clearly expresses what needs to be said.